lib/table.css

```diff
@@ -0,0 +1,271 @@
+/*
+ * Keep in sync with https://github.com/cockpit-project/cockpit/tree/master/pkg/lib/table.css
```

Comment:
I prefer to not to write this header - as it makes sync a that bit more difficult :)

Reply:
ack I plan anyway to make sync automatic through a makefile rule

Reply:
That would be cool :) I just do `cp ../cockpit/pkg/lib/* lib` and then `git add -u`, but rule that would copy just the existing files would be great :)